### PR TITLE
Fix packaging issues

### PR DESCRIPTION
**Pull request by laborleben**
- tested on Windows and Linux
- tested ```pip install``` and ```python setup.py```

### CODE REVIEW

**Comment by laborleben** [comment]
@DavidLP I restarted the build and now it is OK. Was a travis/networking issue before.

File: pixel_clusterizer/clusterizer.py [hunk 1]
```diff
@@ -255,12 +255,14 @@ def set_end_of_cluster_function(self, function):
...
         This is maybe the only way to make the clusterizer to work with multiprocessing.
         '''
+        self.cluster_functions._end_of_cluster_function = self._jitted(function)
         self._end_of_cluster_function = function
 
     def set_end_of_event_function(self, function):
         ''' Adding function to module.
         This is maybe the only way to make the clusterizer to work with multiprocessing.
         '''
+        self.cluster_functions._end_of_event_function = self._jitted(function)
```
Comment (DavidLP) [inline pixel_clusterizer/clusterizer.py]: Why is this needed if the function is nevertheless jitted on each `cluster_hits` call?

Comment (laborleben) [reply pixel_clusterizer/clusterizer.py]: I'd like to know that either, but this was the only solution I found that works on all platforms and installation methods.

To be clear, the function has to jitted twice -- outside of the thread and inside. This is how we get it running on all platforms. This is exactly the same solution you provided with testbeam_analysis in the past but now it is included into pixel_clusterizer.

Comment (DavidLP) [reply pixel_clusterizer/clusterizer.py]: Ok.

**Comment by laborleben** [comment]
The pull request went onto master. We need to fix that.

**Comment by DavidLP** [comment]
Ahhh

**Comment by laborleben** [comment]
Done :-)
